### PR TITLE
SAK-44061: SAMIGO - Error while saving without From(Required) 'Feedback will be displayed to the student on specific dates'

### PR DESCRIPTION
**Pull request by FerAlvarez95**

### CODE REVIEW

**Review by jesusmmp — APPROVED** [review]
LGTM!

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SavePublishedSettingsListener.java [hunk 1]
```diff
@@ -497,7 +497,7 @@ else if(!assessmentSettings.getIsValidFeedbackDate()){
 				error=true;
 			}
 
-			if(StringUtils.isNotBlank(assessmentSettings.getFeedbackEndDateString()) && assessmentSettings.getFeedbackDate().after(assessmentSettings.getFeedbackEndDate())){
+			if(assessmentSettings.getFeedbackDate() == null && StringUtils.isNotBlank(assessmentSettings.getFeedbackEndDateString()) && assessmentSettings.getFeedbackDate().after(assessmentSettings.getFeedbackEndDate())){
```
Comment (adrianfish) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SavePublishedSettingsListener.java:500]: This doesn't make sense to me. Why check if the date is null and then use that date in the after call?

**Review by adrianfish — CHANGES_REQUESTED** [review]
See my comment. I can't tell if I'm losing my eyesight, but those tests look wrong to me.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SavePublishedSettingsListener.java (hunk 1 above)
Comment (FerAlvarez95) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/SavePublishedSettingsListener.java:500]: The error on Jira is caused by a null pointer as feedbackDate, so i think that we need to ensure that it is not null before access it.

**Comment by jesusmmp** [comment]
Please don't merge yet. Tested on my local instance and it doesn't work.

**Comment by jesusmmp** [comment]
@adrianfish @FerAlvarez95 The new PR is here: https://github.com/sakaiproject/sakai/pull/8500

**Pull request closed without merging**